imod/tests/fixtures/mf6_circle_fixture.py

```diff
@@ -340,3 +340,85 @@ def circle_model_transport():
     simtimes = pd.date_range(start="2000-01-01", end="2001-01-01", freq="W")
     simulation.create_time_discretization(additional_times=simtimes)
     return simulation
+
+
+
+@pytest.fixture(scope="function")
+def circle_model_transport_multispecies():
+
+    al = 0.001
+    porosity = 0.3
+    max_concentration = 35.0
+    min_concentration = 0.0
+    max_density = 1025.0
+    min_density = 1000.0
+    species = ["sali", "tempe"]
```

Comment:
Maybe this is a bit better:

```suggestion
    species = ["salt", "temp"]
```